### PR TITLE
Fix: Fix auto-generated docs

### PR DESCRIPTION
**Pull request by josemarinas**
## Description

Fix auto generated docs

Task: [APP-1205](https://aragonassociation.atlassian.net/browse/APP-1205)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder of the package after the [UPCOMING] title and before the latest version.
- [ ] I have tested my code on the test network.

### CODE REVIEW

File: modules/client/src/addressList/internal/client/decoding.ts [hunk 1]
```diff
@@ -10,7 +10,9 @@ import {
 import { AVAILABLE_FUNCTION_SIGNATURES } from "../constants";
 import { IClientAddressListDecoding } from "../../interfaces";
 import { WhitelistVoting__factory } from "@aragon/core-contracts-ethers";
-
+/**
```
Comment (brickpop) [inline modules/client/src/addressList/internal/client/decoding.ts:14]:
```suggestion

/**
```

File: modules/client/src/addressList/internal/client/encoding.ts [hunk 2]
```diff
@@ -16,6 +16,9 @@ import {
 import { WhitelistVoting__factory } from "@aragon/core-contracts-ethers";
 import { addressListInitParamsToContract } from "../utils";
 
+/**
+ * Encoding module the SDK Address List Client
```
Comment (brickpop) [inline modules/client/src/addressList/internal/client/encoding.ts]:
```suggestion
 * Encoding module for the SDK AddressList Client
```

File: modules/client/src/addressList/internal/client/decoding.ts [hunk 3]
```diff
@@ -10,7 +10,9 @@ import {
...
+ * Decoding module the SDK Address List Client
```
Comment (brickpop) [inline modules/client/src/addressList/internal/client/decoding.ts]:
```suggestion
 * Decoding module for the SDK AddressList Client
```

File: modules/client/src/addressList/internal/client/estimation.ts [hunk 4]
```diff
@@ -9,6 +9,9 @@ import {
 } from "../../../client-common";
 import { IClientAddressListEstimation } from "../../interfaces";
 
+/**
+ * Estimation module the SDK Address List Client
```
Comment (brickpop) [inline modules/client/src/addressList/internal/client/estimation.ts:13]: same

File: package.json [hunk 5]
```diff
@@ -21,7 +21,9 @@
     "lint": "tsdx lint modules/*/src",
     "build": "yarn turbo run build",
     "test": "TESTING=true tsdx test --verbose --runInBand",
-    "docs:client": "jsdoc2md --files ./modules/client/src/*.ts --configure ./jsdoc2md.json > ./docs/client.md"
+    "docs:client": "jsdoc2md --files ./modules/client/src/*.ts ./modules/client/src/internal/client/*.ts --configure ./jsdoc2md.json > ./docs/client.md",
+    "docs:addressList": "jsdoc2md --files ./modules/client/src/addressList/*.ts ./modules/client/src/addressList/internal/client/*.ts --configure ./jsdoc2md.json > ./docs/addressList.md",
+    "docs:erc20": "jsdoc2md --files ./modules/client/src/erc20/*.ts ./modules/client/src/erc20/internal/client/*.ts --configure ./jsdoc2md.json > ./docs/erc20.md"
```
Comment (brickpop) [inline package.json:26]: That might be OK, but we are documenting the `internal` folder source files. That might be an issue if people start to depend on this

Comment (josemarinas) [reply package.json:26]: The solution might be moving the comments from the class to the interface in `interfaces.ts`, this is out of the `internal` folder

**Review by brickpop — APPROVED** [review]

**Pull request merged**